core/services/postgres/sqlx.go

```diff
@@ -0,0 +1,89 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/jmoiron/sqlx/reflectx"
+	mapper "github.com/scylladb/go-reflectx"
+	"github.com/smartcontractkit/sqlx"
+	"gorm.io/gorm"
+)
+
+type Queryer interface {
+	sqlx.Ext
+	sqlx.ExtContext
+	QueryRow(query string, args ...interface{}) *sql.Row
+}
+
+func WrapDbWithSqlx(rdb *sql.DB) *sqlx.DB {
+	db := sqlx.NewDb(rdb, "postgres")
+	db.MapperFunc(mapper.CamelToSnakeASCII)
+	return db
+}
+
+func UnwrapGormDB(db *gorm.DB) *sqlx.DB {
+	d, err := db.DB()
+	if err != nil {
+		panic(err)
```

Comment:
is it ok to panic here? could use a comment if yes

Reply:
This is only an error if `gorm.DB` is completely invalid, i.e. if you construct it by hand (`gorm.DB{}`). It's annoying to have to deal with this `err` everywhere so I figured it's OK to panic.